plinth/modules/users/__init__.py

```diff
@@ -86,11 +90,24 @@ def _diagnose_ldap_entry(search_item):
             .format(search_item=search_item), result]
 
 
-def add_group(group):
+def create_group(group):
     """Add an LDAP group."""
     actions.superuser_run('users', options=['create-group', group])
```

Comment:
When I install an app, I don't see the group show up on the Edit User page, until plinth is restarted.

Reply:
I think this one can be fixed by adding a call to register_group(group).